<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictionaryForm</class>
 <widget class="QWidget" name="DictionaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_back">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../icons.qrc">
         <normaloff>:/icons/16x16/go-previous.png</normaloff>:/icons/16x16/go-previous.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_forward">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../icons.qrc">
         <normaloff>:/icons/16x16/go-next.png</normaloff>:/icons/16x16/go-next.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_openIn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../icons.qrc">
         <normaloff>:/icons/16x16/text-html.png</normaloff>:/icons/16x16/text-html.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_input"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_openInBrowser">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../icons.qrc">
         <normaloff>:/icons/16x16/internet-web-browser.png</normaloff>:/icons/16x16/internet-web-browser.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_enter">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../icons.qrc">
         <normaloff>:/icons/16x16/go-jump-locationbar.png</normaloff>:/icons/16x16/go-jump-locationbar.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListView" name="listView"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
